Flight Software/QT RP2040/Velocity data/code.py

```diff
@@ -34,13 +32,13 @@
 initial_time = time.monotonic()
 vel_0 = 0
 time_0 = 0
-accel_0 = 0
+accel_0 = g
 
 # Ascent mode
 while True: #accel_mag > 30 or v0 > 10:
     icm_accel = icm.acceleration
     current_time = time.monotonic()
-    time_stamp = current_time - initial_time
+    time_1 = current_time - initial_time
```

Comment:
Should probably stay as time_stamp, makes it more clear what is or something idkr